File: python/P6.py
def solve(N):
	elements = [x for x in range(1,N+1)]
	num = len(elements)
	i = 0
	SUM = 0
	while i < num - 1:
		j = i + 1
		while j < num:
			SUM += 2 * elements[i] * elements[j]
			j += 1
		i += 1
	return SUM

if __name__ == "__main__":
    print(solve(100))

File: java/P6.java
import java.util.*;
import java.util.stream.IntStream;

public class P6 {
    public static void main(String[] args) {
        System.out.println(solve(100));
    }

    private static int solve(int N) {
        int[] elements = IntStream.range(1, N).toArray();
        int i = 0;
        int SUM = 0;
        while (i < elements.length - 1) {
            int j = i + 1;
            while (j < elements.length) {
                SUM += 2 * elements[i] * elements[j];
                j += 1;
            }
            i += 1;
        }
        return SUM;
    }
}